.gitignore

```diff
@@ -11,4 +11,4 @@ resources/dark
 resources/martor-description.css
 resources/style.css
 resources/vars.scss
-sass_processed
+sass_processed
```

Comment:
Can we not change this file at all?

Reply:
I apologize, downloading the original file again.